core/common/src/main/java/alluxio/wire/WorkerInfo.java

```diff
@@ -190,6 +191,22 @@ public boolean equals(Object o) {
         && mStartTimeMs == that.mStartTimeMs;
   }
 
+  /**
+   * Uses the last contact seconds of WorkerInfo o.getLastContactSec() to determine order
```

Comment:
- Avoid referring to implementation details like `o.getLastContactSec()` in javadoc. 
- It isn't clear what "newest on top" means - you could say something like "from most recently contacted to least recently contacted"